tests/testthat/test-pick_unmatched.R

```diff
@@ -0,0 +1,59 @@
+test_that("if `accept_match` is a character errors gracefully (#122)", {
```

Comment:
What this function actually does is now clear from the test titles.
You can see them all [here](https://github.com/2DegreesInvesting/tilt.company.match/blob/dcd3adfd960550ea7061c8fd7c37d6837ee3ca48/tests/testthat/test-pick_unmatched.R)